mlflow/utils/databricks_utils.py

```diff
@@ -22,6 +23,7 @@
 
 _logger = logging.getLogger(__name__)
 
+MODEL_DEPENDENCY_OAUTH_TOKEN_FILE_PATH = "/var/credentials-secret/model-dependency-token"
```

Comment:
Since this is only used in `_get_model_dependency_oauth_token`, can we move it inside that method?